### PR TITLE
Copyright to Cognite

### PR DESCRIPTION
**Pull request by eventh**
Signed-off-by: Even Wiik Thomassen <even.thomassen@cognite.com>

### CODE REVIEW

**Comment by wjoel** [comment]
Should it be "Cognite" or "Cognite AS" for those things?

**Comment by eventh** [comment]
I used the same as the homepage, just Cognite :)

File: docs/source/conf.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
 # General information about the project.
 project = "cognite-sdk"
-copyright = "2018, Erlend Vollset"
+copyright = "2019, Cognite"
```
Comment (wjoel) [inline docs/source/conf.py]: According to https://cognitedata.atlassian.net/wiki/spaces/COG/pages/7143447/Open+Source+Policy it should be "Cognite AS"

Comment (eventh) [reply docs/source/conf.py]: Fixed

**Pull request merged**